docs/faq.md

```diff
@@ -1,8 +1,14 @@
-# Frequenly answered questions
+# Frequently Answered Questions
```

Comment:
Our titles on all pages are capitalized with only the first word right now. If we want to make this change, I'd like it to be all or nothing.

For this PR, we can probably leave it the same.

Reply:
I think page titles would make sense to be in Pascal Case. Especially when we're using acronyms like FAQ. 

It should also help make a logical distinction between pages and subpages (which could be sentences) if we ever needed to break out a dedicated page for a particular entry, but still wanted to keep it listed in its FAQ section.

Reply:
I'm down to change it 👍 But we should do it for all the pages at once for the sake of consistency.

Reply:
Also - Is it "Frequently Asked Questions" or "Frequently Answered Questions"?